hack/print-workspace-status.sh

```diff
@@ -35,9 +35,9 @@ STABLE_BUILD_SCM_STATUS ${KUBE_GIT_TREE_STATE-}
 STABLE_BUILD_SCM_REVISION ${KUBE_GIT_VERSION-}
 STABLE_BUILD_MAJOR_VERSION ${KUBE_GIT_MAJOR-}
 STABLE_BUILD_MINOR_VERSION ${KUBE_GIT_MINOR-}
-STABLE_gitCommit ${KUBE_GIT_COMMIT-}
-STABLE_gitTreeState ${KUBE_GIT_TREE_STATE-}
-STABLE_gitVersion ${KUBE_GIT_VERSION-}
-STABLE_gitMajor ${KUBE_GIT_MAJOR-}
-STABLE_gitMinor ${KUBE_GIT_MINOR-}
+gitCommit ${KUBE_GIT_COMMIT-}
+gitTreeState ${KUBE_GIT_TREE_STATE-}
+gitVersion ${KUBE_GIT_VERSION-}
+gitMajor ${KUBE_GIT_MAJOR-}
+gitMinor ${KUBE_GIT_MINOR-}
```

Comment:
maybe add `buildDate $(date -u +'%Y-%m-%dT%H:%M:%SZ')`?

```console
$ ./kubernetes/client/bin/kubectl version --client
Client Version: version.Info{Major:"1", Minor:"7+", GitVersion:"v1.7.0-alpha.0.2075+4399de75c61445", GitCommit:"4399de75c61445a357e0a8093fcdfcf16c18cfee", GitTreeState:"clean", BuildDate:"1970-01-01T00:00:00Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
```

Bazel auto-generates `BUILD_TIMESTAMP`, but that won't work for our linking rule.